src/components/Search/SearchComponent.tsx

```diff
@@ -101,12 +96,13 @@ function SearchComponent({
           </Tooltip>
         </InputLabel>
         {/**** Edit Button ****/}
-        {hasNonEditableState && !formIsDisplayed && (
+        {/* temporary until global edit button PR */}
+        {hasNonEditableState && !formIsDisplayed && isBaseComp && (
           <EditButton
             isBase={isBaseComp}
             onEditAction={() => {
               onEdit();
-              setFormIsDisplayed(true);
+              onSetDisplayForm(true);
```

Comment:
If the state goes up, I think this could go in the onEdit callback (in the caller) directly, and we don't need `onSetDisplayForm` at all.